flytekit/extend/backend/utils.py

```diff
@@ -20,13 +20,15 @@ def convert_to_flyte_phase(state: str) -> TaskExecution.Phase:
     Convert the state from the agent to the phase in flyte.
     """
     state = state.lower()
-    # timedout is the state of Databricks job. https://docs.databricks.com/en/workflows/jobs/jobs-2.0-api.html#runresultstate
-    if state in ["failed", "timeout", "timedout", "canceled"]:
+    # timedout, skipped, internal_error, terminating and pending is the state of a Databricks job. https://docs.databricks.com/en/workflows/jobs/jobs-2.0-api.html#runresultstate
```

Comment:
we can remove this, many platforms have these states
```suggestion
```

Reply:
no problem, thank you